docker-compose-develop.yml

```diff
@@ -59,11 +59,8 @@ services:
       - predictions:/data/prankweb
 volumes:
   rabbitmq:
-    external: True
```

Comment:
What is the objective of this change? The idea behind using external volumes is that a user should create them with proper mapping on their system. By removing the external I believe docker-compose may create a custom volumes. The question is what is the expected behavior.

Is this documented in the wiki?

Reply:
If an user follows the current wiki tutorial for Docker setup, there are no instructions for any mapping. So, when the user tries to run docker-compose, the following message appears for all three volumes:

`ERROR: Volume prankweb_rabbitmq declared as external, but could not be found. Please create the volume manually using `docker volume create --name=prankweb_rabbitmq` and try again.`

That means the user has to create the volume themselves with the mentioned command, but as there are no steps how to create you own mapping, I felt like there is no reason to create it manually and it would be easier to let docker-compose do it instead. The behavior was the same.

If there is any reason user should create the volumes in any other way, this change may not be needed, but then the wiki should contain information about creation (and possible mapping) of the volumes.

Reply:
I would update the wiki and keep the external in the compose file. I agree that it may not be needed for all volumes like prankweb_rabbitmq. The objective is to prevent situation where the predictions and the conservation files are part of a created volume without user knowing.  The reason is that the conservation volume can be quite large ( 20GB ).

Reply:
Sounds reasonable, I changed the volumes back to their original state in the last commit. I will create an issue in the wiki repository, so we do not forget to update it.